packages/common/src/loader.ts

```diff
@@ -14,10 +14,27 @@ export class Loader {
   // Returns true if ENgrid should reload (that means the current ENgrid is not the right one)
   // Returns false if ENgrid should not reload (that means the current ENgrid is the right one)
   public reload() {
+    const assets = this.getOption("assets");
     const isLoaded = ENGrid.getBodyData("loaded");
-    let assets = this.getOption("assets");
+    const shouldSkipCss = this.getOption("engridcss") === "false";
+    const shouldSkipJs = this.getOption("engridjs") === "false";
 
     if (isLoaded || !assets) {
+      if (shouldSkipCss && this.cssElement) {
+        this.logger.log("engridcss=false | Removing original stylesheet:", this.cssElement);
+        this.cssElement.remove();
+      }
+
+      if (shouldSkipJs && this.jsElement) {
+        this.logger.log("engridjs=false | Removing original script:", this.jsElement);
+        this.jsElement.remove();
+      }
+
+      if (shouldSkipJs) {
+        this.logger.log("engridjs=false | Skipping JS load.");
+        return true;
```

Comment:
@fernanDOTdo This bit I'm not sure about, but I think it has the desired behavior. From the task description:

> Then, for this example: ?assets=digital-wallets&engridjs=false the loader will load the code via JSDelivr for the digital-wallets branch but will remove the JS and prevent it from loading.

My understanding is we don't want any ENGrid JS running on the page when `engridjs=false`, and returning `true` here is the only way I could find to stop that from happening.

Reply:
It's perfect